src/index.html

```diff
@@ -7,7 +7,54 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="page__body">
```

Comment:
no needs to add a class selector to a body since you have only one body on the page